diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/AvoidUtilityClass.kt

```diff
@@ -36,8 +37,12 @@ class AvoidUtilityClass(private val configRules: List<RulesConfig>) : Rule("avoi
     override fun visit(node: ASTNode, autoCorrect: Boolean, emit: EmitType) {
         emitWarn = emit
         isFixMode = autoCorrect
-        if (node.elementType == OBJECT_DECLARATION || node.elementType == CLASS ) {
-            checkClass(node)
+        val config = configRules.getCommonConfiguration().value
```

Comment:
```suggestion
        val config by configRules.getCommonConfiguration()
```
You can use it as a delegate here